views/celebrities/edit.hbs

```diff
@@ -0,0 +1,8 @@
+<h2>Edit celebrity</h2>
+<form action="/celebrities/{{celebrity._id}}" method="post">
+
+Name: <input type="text" name="name" value="{{celebrity.name}}"><br>
```

Comment:
What are these?? `Name` `Occupation` and `Phrase`, they look like labels? Try to use the correct tag.
And try not to use `<br>`